README.md

```diff
@@ -207,9 +203,7 @@ For more instructions on how to install the `:fast_tls` dependency, please visit
 
 #### Why, after joining the room, I don't see a video stream from the others?
 That might be due to the misconfiguration of the `EXTERNAL_IP` environment variable. Make sure, that the variable is set in the environment where you are running the server, as well as that it is the IPv4 address pointing to the server, visible by all the peers.
-
-#### When I run the videoroom in the docker container, why cannot I access VP8 tracking?
-You need to publish docker ports used for sending and receiving media (`INTEGRATED_TURN_PORT_RANGE` environment variable) or you can use the host network if you are running on Linux.
+Please also keep in mind that you need to publish docker ports used for sending and receiving media (`INTEGRATED_TURN_PORT_RANGE` environment variable) or use the host network if you are running on Linux.
```

Comment:
Take into mind, that the port number stored in `INTEGRATED_TCP_TURN_PORT` has to be published as well. You can also add information, that `INTEGRATED_TURN_PORT_RANGE` is a range of published ***UDP*** ports, because in Docker publishing ports by default works only on TCP